terraform/groups/lambda/module-lambda/main.tf

```diff
@@ -0,0 +1,40 @@
+data "vault_generic_secret" "lambda_environment_variables" {
+  path = "applications/${var.aws_profile}/${var.environment}/${var.service}/lambda_environment_variables"
+}
+
+locals {
+  sftp_port = 22
```

Comment:
If these are only used in one place I'd just get rid of them and code the value directly

Reply:
OK, removed the locals block and added values in-line.